src/accordion/_base.scss

```diff
@@ -25,8 +30,11 @@
 
 .accordion-section-toggle {
   cursor: pointer;
+  padding: ms(1) 0;
+  margin-bottom: 0;
 
   [data-icon="arrow"] {
+    float: right;
```

Comment:
@beckichoi / @ckeisato: 

The target to open and close the accordion needs to account for variable text length. In its currently implementation, a question has a really long title  will eventually wrap and the alignment with the arrow will be off especially on small devices. 

My recommendation would be to use flexbox to layout the `accordion-section-toggle`—one area of title, one area for the icon. Design will also want to set rules for length of text to prevent odd text on small devices. Or perhaps we'll want to use ellipsis...There are options for you both to explore during fit and finish! 

@ArmandoAmador Can you run some queries for us? We are looking for 2-5 examples FAQs with really long titles and a 2-5 examples of short to ave length. At the moment do we limit the length of the title in the CMS?